src/blockchain/blockchain.service.ts

```diff
@@ -135,4 +135,8 @@ export class BlockchainService implements OnApplicationBootstrap, OnApplicationS
     const epochLength: u32 = await this.query('capacity', 'epochLength');
     return typeof epochLength === 'number' ? epochLength : epochLength.toNumber();
   }
+
+  public async getNonce(account: Uint8Array): Promise<number> {
+    return this.rpc('system', 'accountNextIndex', account);
+  }
```

Comment:
I think `accountNextIndex` returns an `Index`, which is a `u32`, not a JS `number`

Reply:
I think the wrappers internally be calling `toNumber` before returning, assuming we would never reach a bigint


Do we want to make it bigint?